tests/functional/u/unnecessary/unnecessary_dict_index_lookup.txt

```diff
@@ -1,23 +1,24 @@
-unnecessary-dict-index-lookup:7:10:7:19::Unnecessary dictionary index lookup, use 'v' instead:UNDEFINED
-unnecessary-dict-index-lookup:19:36:19:45::Unnecessary dictionary index lookup, use 'v' instead:UNDEFINED
-unnecessary-dict-index-lookup:21:1:21:10::Unnecessary dictionary index lookup, use 'v' instead:UNDEFINED
-unnecessary-dict-index-lookup:22:1:22:10::Unnecessary dictionary index lookup, use 'v' instead:UNDEFINED
-unnecessary-dict-index-lookup:22:44:22:53::Unnecessary dictionary index lookup, use 'v' instead:UNDEFINED
-unnecessary-dict-index-lookup:24:33:24:42::Unnecessary dictionary index lookup, use 'v' instead:UNDEFINED
-unnecessary-dict-index-lookup:26:1:26:10::Unnecessary dictionary index lookup, use 'v' instead:UNDEFINED
-unnecessary-dict-index-lookup:27:1:27:10::Unnecessary dictionary index lookup, use 'v' instead:UNDEFINED
-unnecessary-dict-index-lookup:27:41:27:50::Unnecessary dictionary index lookup, use 'v' instead:UNDEFINED
-unnecessary-dict-index-lookup:37:10:37:23::Unnecessary dictionary index lookup, use 'v' instead:UNDEFINED
-unnecessary-dict-index-lookup:39:19:39:32::Unnecessary dictionary index lookup, use 'v' instead:UNDEFINED
-unnecessary-dict-index-lookup:43:40:43:53::Unnecessary dictionary index lookup, use 'v' instead:UNDEFINED
-unnecessary-dict-index-lookup:45:1:45:14::Unnecessary dictionary index lookup, use 'v' instead:UNDEFINED
-unnecessary-dict-index-lookup:46:1:46:14::Unnecessary dictionary index lookup, use 'v' instead:UNDEFINED
-unnecessary-dict-index-lookup:46:52:46:65::Unnecessary dictionary index lookup, use 'v' instead:UNDEFINED
-unnecessary-dict-index-lookup:48:37:48:50::Unnecessary dictionary index lookup, use 'v' instead:UNDEFINED
-unnecessary-dict-index-lookup:50:1:50:14::Unnecessary dictionary index lookup, use 'v' instead:UNDEFINED
-unnecessary-dict-index-lookup:51:1:51:14::Unnecessary dictionary index lookup, use 'v' instead:UNDEFINED
-unnecessary-dict-index-lookup:51:49:51:62::Unnecessary dictionary index lookup, use 'v' instead:UNDEFINED
-unnecessary-dict-index-lookup:57:10:57:20::Unnecessary dictionary index lookup, use 'item[1]' instead:UNDEFINED
-unnecessary-dict-index-lookup:60:1:60:11::Unnecessary dictionary index lookup, use 'item[1]' instead:UNDEFINED
-unnecessary-dict-index-lookup:65:10:65:20::Unnecessary dictionary index lookup, use 'item[1]' instead:UNDEFINED
-unnecessary-dict-index-lookup:82:14:82:18::Unnecessary dictionary index lookup, use '_' instead:UNDEFINED
+unnecessary-dict-index-lookup:7:10:None:None::Unnecessary dictionary index lookup, use 'v' instead:UNDEFINED
```

Comment:
Are you by any chance working on Python < 3.8? 
The end line and end column information of these messages should be added here, they seem to have been transformed into `None` due to the auto-updater. This is probably also going to fail the test.

@Pierre-Sassoulas Shall we make the updater only accessible on > 3.7? Or make it put out a warning on lower versions?

Reply:
Yes we need a run time error if the upgrade would be useless in CI. #5418 is ideal for that I guess ?

Reply:
Yeah! Seems like a good idea. For now `3.8` is good enough since we don't have any nodes that only have their `end_column` data on 3.9 or 3.10. I think an exception when trying to run and a small explanation on why 3.8 is necessary should do.

Reply:
https://github.com/PyCQA/pylint/pull/5418/commits/1e29198d93f82392b0154ad87908853acde9f0ee

Reply:
my bad let me switch my environments